src/App.css

```diff
@@ -207,19 +228,30 @@ textarea {
 
 #description-wrapper {
   overflow: auto;
-  height: calc((75vh - 178px)/2);
+  height: calc((75vh - 178px) / 2);
   font-size: 80%;
 }
 
-#selected-request {
-  font-size: 150%;
+#selected-request,
+#message-container {
+  font-size: 130%;
   color: #606060;
   padding: 7px;
 }
 
-#contact-button{
+#offerMessage {
+  font-size: 90%;
+  color: #606060;
+  padding: 10px;
+}
+#send-message-form {
+  padding: 10px;
+  padding-top: 7px;
+}
+#contact-button,
+#message-send-btn {
   position: absolute;
-  bottom: 39px;
+  bottom: 15px;
```

Comment:
you should prob think about writing it like this:
bottom: 
![image](https://user-images.githubusercontent.com/59050770/85251487-af0a5380-b459-11ea-9269-a90b1489fce9.png) ;

Reply:
haha, could u help me?